rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java

```diff
@@ -531,6 +533,22 @@ public final void internalShutDown() {
      */
     protected abstract void configure(Object... parameters);
 
+    /**
+     * Reconfigure the infrastructure of a potentially already deployed node
+     * source with the given parameters.
+     * Implementations are free to handle override of parameters as they wish.
+     *
+     * @see Configurable#dynamic()
+     *
+     * @param updatedInfrastructureParameters parameters potentially containing
+     *                                    updated dynamic parameters
+     *
+     * @throws IllegalArgumentException if parameters are incorrect
+     */
+    public void reconfigure(Object... updatedInfrastructureParameters) {
+
```

Comment:
add a small comment in the method block, saying why this method is empty

Reply:
Done.